docs/src/release_notes.md

```diff
@@ -1,6 +1,54 @@
 # Release notes
 
-## Version 0.22.3 (February 10, 2021)
+## Version 0.23.0 (February 25, 2022)
+
+**JuMP v0.23.0 is a breaking release. It is also a release-candidate for JuMP
+v1.0.0. That is, if no issues are found with the v0.23.0 release, then it will
+be re-tagged as v1.0.0.**
+
+- Breaking changes:
```

Comment:
Should we mention the switch to MOI 1.0?

Reply:
Done.